llvm/lib/Target/RISCV/RISCVSchedSyntacoreSCR7.td

```diff
@@ -0,0 +1,378 @@
+//==- RISCVSchedSyntacoreSCR7.td - Syntacore SCR7 Sched Defs -*- tablegen -*-=//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+//===----------------------------------------------------------------------===//
+
+// This file covers scheduling model for rv64imafdc_zba_zbb_zbc_zbs
```

Comment:
According to 108406 I think this processor is rv64imafdcv_zba_zbb_zbc_zbs_zkn. Are we missing `v` and `zkn` here?

Reply:
I think it would be good to add a FIXME here to add `v` and `zkn` here.

Reply:
addressed